src/main.py

```diff
@@ -34,6 +36,8 @@
 @bot.event
 async def on_ready():
 	log.info(f'Logged in as {bot.user}')
+	ending_note = 'Powered by ' + bot.user.name + '\nFor command {help.clean_prefix}{help.invoked_with}'
```

Comment:
Fixed my f-string suggestion to work with the string formatting that the help library applies later

```suggestion
	ending_note = f'Powered by {bot.user.name}\nFor command {{help.clean_prefix}}{{help.invoked_with}}'
```